distribution/openrct2.d.ts

```diff
@@ -2606,6 +2606,56 @@ declare global {
         "waiting_to_depart" |
         "waiting_to_start";
 
+    type CrashParticleType =  0 | 1 | 2 | 3 | 4;
+
+    /**
+     * Represents a vehicle explosion particle. They are emitted during a vehicle
+     * crash and will bounce until their timer expires and they are automatically
+     * removed.
+     */
+    interface CrashedVehicleParticle extends Entity {
+        /**
+         * The colour of the particle. Only body and trim are used.
+         */
+        colours: VehicleColour;
```

Comment:
`VehicleColour.tertiary` does not seem to be doing anything. Maybe we can introduce a different interface to avoid confusion about this:
```ts
interface CrashedVehicleColour {
        body: number;
        trim: number;
}
```

Reply:
I don't use ts transpilers so I don't know how ts users feel about it, but a new interface just for "VehicleColour but only body and trim are used" seems like it would add complexity for not a whole lot. As-is, it's just like any vehicle that only uses 2 colours.

Reply:
If it is to be changed to its own interface, the play to me seems to be to create an interface e.g. `VehicleColourSimple` that has body and trim, then extend that interface as `VehicleColour`. But if you prefer it as its own separate interface called `CrashedVehicleColour` then I will do that.

Reply:
In Typescript interfaces are basically air and are meant for type hints. It's whole purpose is to tell users what are valid properties and what are not. If the interface type cannot hint that `tertiary` is not valid, then what else can we not trust about it? 🙂

Also interfaces can map to another as long as their properties match, e.g. this is allowed:
```ts
var particleColours: CrashedVehicleColour = <VehicleColour>car.colours;
```

I noticed the issue here because I was setting the particle two 3 different colours, but when getting the colours that tertiary colour is not returned anymore, which made it a really confusing side effect.

Reply:
@at your last comment: I'd prefer `CrashedVehicleColour` as it's named more explicit. As mentioned, since interfaces are basically air and can map to another, it doesn't matter if there's similarities. There's not really any benefit from inheritance if it's only a few properties.